linux_pipeline/Jenkinsfile_developer_patch_validation

```diff
@@ -194,11 +196,53 @@ pipeline {
                     }
                     agent {
                         node {
-                            label 'meta_slave'
+                            label 'hyper-v'
                         }
                     }
                     steps {
-                        echo "NOOP Lisa test."
+                        withCredentials(bindings: [string(credentialsId: 'WIN_SMB_SHARE_URL', variable: 'SMB_SHARE_URL'),
+                                                   string(credentialsId: 'LISA_IMAGES_SHARE_URL', variable: 'LISA_IMAGES_SHARE_URL'),
+                                                   string(credentialsId: 'AZURE_SAS', variable: 'AZURE_SAS'),
+                                                   string(credentialsId: 'AZURE_STORAGE_URL', variable: 'AZURE_STORAGE_URL'),
+                                                   usernamePassword(credentialsId: 'smb_share_user_pass',
+                                                                passwordVariable: 'PASSWORD',
+                                                                usernameVariable: 'USERNAME')]) {
+                            echo 'Running LISA...'
+                            dir('kernel_version' + env.BUILD_NUMBER + env.BRANCH_NAME) {
+                                PowerShellWrapper('New-Item -Type Directory scripts/package_building')
```

Comment:
here you need to revert to unstash kernel_ini like in here

Reply:
https://github.com/LIS/lis-pipeline/pull/127/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3L293

Reply:
you need to remove the PowerShell wrappers here, as they are not valid anymore.